test/libyul/yulOptimizerTests/controlFlowSimplifier/switch_remove_empty_case.yul

```diff
@@ -8,9 +8,11 @@
 // ----
 // step: controlFlowSimplifier
 //
-// {
-//     let y := 200
-//     switch calldataload(0)
-//     case 1 { y := 9 }
-//     case 2 { y := 10 }
+// object "object" {
```

Comment:
Did not update every test, but this is what it looks like. Majority (90%) of the tests do no have objects, but those which do result in a very weird uncompilable output.

We could also try to be smart and use code printing if there was only a code object and nothing.